examples/OSPolicyAssignments/console/win-restore-rdp-powershell.yaml

```diff
@@ -0,0 +1,142 @@
+id: win-policy
+mode: VALIDATION
+resourceGroups: 
+  - resources: 
+      exec: 
+        validate: 
```

Comment:
Hi @adjackura - Thanks for the feedback. I've separated the logic between validate and enforce statement. 

Unfortunately, it doesnt seem to [parse the exit 101 code](https://github.com/GoogleCloudPlatform/osconfig/pull/393/commits/e31e295aa46ef48f411536d316f4059a0056339e#diff-f82457b4e745577b1576e318b89ee709ea11a7bee71f675ddfcb57ff6231638eR156) to trigger the enforce steps. I can verify that the script runs successfully within the OS via C:\transcript_validate.log.

Is there any way I can debug what the exit code is actually being passed to OS guest policy (or whatever is invoking the inline script)?

Reply:
so your saying validate always returns 100 and doesn't trigger enforce?

If you run with this metadata key and check cloud logs you should see the agent walk through the entire policy application with exit codes:
osconfig-log-level=debug

you can see our testing example (its in code not in yaml), but I dont see anything obviously wrong with your logic, my best guess is that it returns 100 each time because of some bug in the check logic:
https://github.com/GoogleCloudPlatform/osconfig/blob/master/e2e_tests/test_suites/ospolicies/ospolicies_test_data.go#L1667